src/main/java/com/zufar/onlinestore/product/dto/ProductInfoRequestResponseDto.java

```diff
@@ -0,0 +1,28 @@
+package com.zufar.onlinestore.product.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import java.util.UUID;
+
+public record ProductInfoRequestResponseDto(
```

Comment:
rename this ugly class name, please 
`ProductDto` looks better and more clear

Reply:
Fixed